racket-util.el

```diff
@@ -118,7 +118,9 @@ The \"project\" is determined by trying, in order:
         (and (fboundp 'vc-root-dir)
              (vc-root-dir))
         (and (fboundp 'project-current)
-             (cdr (project-current nil dir)))
+             (let ((pr (project-current nil dir)))
+               (and (fboundp 'project-root)
```

Comment:
@greghendershott I am not sure this is the right way to make CI happy.

Reply:
I think what we really want here is to load the library `project` on demand if it exists, but I have not figured how to do that properly.